.github/ISSUE_TEMPLATE/development-issue.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Development Issue
+about: 关于技能开发， 语音接入， 智能家居接入， 接入协议相关的问题
+title: ''
+labels: development
```

Comment:
development are more generic, we should describe in modules.

Reply:
yes， you are right, see the previous suggestion

Reply:
After changing to "ask a question", the label should be "question".